public.js

```diff
@@ -253,6 +253,45 @@ var init = function(app, schema, sequelize, config) {
   app.get('/js/:domain*', js_result('success'));
   app.get('/jsaction/:domain*', js_result('action'));
 
+  function json_result (successMetric) {
+    return function (req, res) {
+      if (! (req.params.domain in config.domain_whitelist)) {
+        return res.status(404).send("Not found");
+      }
+      res.setHeader("Cache-Control", "no-cache, no-store, must-revalidate");
+
+      var murl = (req.params.domain + decodeURIComponent(req.params[0] || '/'));
+      bandit.choose(murl, sequelize, successMetric).then(function(trialChoice) {
+        schema.Metadata.findAll({where: {id: trialChoice}, plain:true, attributes: ['id', 'url', 'headline','text','image_url']}).then(function(results) {
+          var r= results.toJSON();
```

Comment:
Linter config is in a new branch here: https://github.com/MoveOnOrg/sharebandit/blob/linter/.eslintrc.json and linted code is in a PR